tests/models/rembert/test_tokenization_rembert.py

```diff
@@ -0,0 +1,162 @@
+        self.assertEqual(vocab_keys[0], "<unk>")
+        self.assertEqual(vocab_keys[1], "<s>")
+
+        self.assertEqual(vocab_keys[5], "▁the")
+        self.assertEqual(vocab_keys[2], "</s>")
+
+    def test_vocab_size(self):
+        self.assertEqual(self.get_tokenizer().vocab_size, 1_000)
+
+    def test_rust_and_python_full_tokenizers(self):
+        if not self.test_rust_tokenizer:
+            return
+
+        tokenizer = self.get_tokenizer()
+        rust_tokenizer = self.get_rust_tokenizer()
+
+        sequence = "I was born in 92000, and this is falsé."
+
+        tokens = tokenizer.tokenize(sequence)
+        rust_tokens = rust_tokenizer.tokenize(sequence)
+        self.assertListEqual(tokens, rust_tokens)
+
+        ids = tokenizer.encode(sequence, add_special_tokens=False)
+        rust_ids = rust_tokenizer.encode(sequence, add_special_tokens=False)
+        self.assertListEqual(ids, rust_ids)
+
+        rust_tokenizer = self.get_rust_tokenizer()
+        ids = tokenizer.encode(sequence)
+        rust_ids = rust_tokenizer.encode(sequence)
+        self.assertListEqual(ids, rust_ids)
+
+    def test_full_tokenizer(self):
+        tokenizer = RemBertTokenizer(SAMPLE_VOCAB, keep_accents=True)
+
+        tokens = tokenizer.tokenize("This is a test")
+        self.assertListEqual(tokens, ["▁This", "▁is", "▁a", "▁t", "est"])
+
+        self.assertListEqual(
+            tokenizer.convert_tokens_to_ids(tokens),
+            [285, 46, 10, 170, 382],
+        )
+
+        tokens = tokenizer.tokenize("I was born in 92000, and this is falsé.")
+        self.assertListEqual(
+            tokens,
+            [
+                SPIECE_UNDERLINE + "I",
+                SPIECE_UNDERLINE + "was",
+                SPIECE_UNDERLINE + "b",
+                "or",
+                "n",
+                SPIECE_UNDERLINE + "in",
+                SPIECE_UNDERLINE + "",
+                "9",
+                "2",
+                "0",
+                "0",
+                "0",
+                ",",
+                SPIECE_UNDERLINE + "and",
+                SPIECE_UNDERLINE + "this",
+                SPIECE_UNDERLINE + "is",
+                SPIECE_UNDERLINE + "f",
+                "al",
+                "s",
+                "é",
+                ".",
+            ],
+        )
+        ids = tokenizer.convert_tokens_to_ids(tokens)
+        self.assertListEqual(ids, [8, 21, 84, 55, 24, 19, 7, 0, 602, 347, 347, 347, 3, 12, 66, 46, 72, 80, 6, 0, 4])
+
+    def test_encode_decode_round_trip(self):
+        tokenizer = RemBertTokenizer(SAMPLE_VOCAB, keep_accents=True)
+        text = "清水寺は京都にある。"
+        ids = tokenizer._tokenize(text)
+        text_decode = tokenizer.convert_tokens_to_string(ids)
+        self.assertEquals(text_decode, text)
+
+        text = "In the sky up above"
+        encode_text = tokenizer._tokenize(text)
+        decode_text = tokenizer.convert_tokens_to_string(encode_text)
+        self.assertEqual(text, decode_text)
+
```

Comment:
let' s try to right the expected raw IDS. Since rembert is a sentencepiece based model we could also add some edge case of what we are expecting

Reply:
Agreed, can you please hint me as to how we get the expected ID and token for the fixtures/test_sentencepiece.model ? 

If I have this information, I can add more cases to the function

Reply:
you can just run the test and print the outputs of `tokenizer._tokenize(text)` which is text, and `tokenizer.encode(text)` which are input ids